README.md

```diff
@@ -15,20 +15,22 @@
 </p>
 
 [Arcjet][arcjet] helps developers protect their apps in just a few lines of
-code. Implement rate limiting, bot protection, email verification & defense
+code. Implement rate limiting, bot protection, email verification, and defend
```

Comment:
"defend" doesn't sound right here. I think it should be "defense". If we read the sentence (without the list) as "Implement defense against common attacks" it sounds correct, while "Implement defend against common attacks" doesn't.